src/main/kotlin/com/openlattice/apps/historical/HistoricalAppTypeSetting.kt

```diff
@@ -0,0 +1,11 @@
+package com.openlattice.apps.historical
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.authorization.Permission
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+data class HistoricalAppTypeSetting(
```

Comment:
What is a historical app type setting for?

Reply:
AppTypeSetting is deleted in the apps v2 branches, so I copied the class here so that we could use it for the migration

Reply:
OH jk I thought this was a different thing in mechanic -- this is something else.

HistoricalAppTypeSetting is here in order to keep the existing API functional so that all the apps don't break before they switch over to the API. Once they've all been updated, we should be able to delete this and delete the current `getAvailableAppConfigs` endpoint.